src/components/pages/telling-story/hero/form/form.jsx

```diff
@@ -0,0 +1,210 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import classNames from 'classnames';
+import { motion } from 'framer-motion';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+
+import Button from 'components/shared/button';
+import Checkbox from 'components/shared/checkbox';
+import Field, { FIELD_TAGS } from 'components/shared/field';
+import Link from 'components/shared/link';
+
+import SuccessHero from './images/success.svg';
+
+const APPEAR_AND_EXIT_ANIMATION_DURATION = 0.5;
+
+const validationSchema = yup.object().shape({
+  firstName: yup.string().trim().required('Name is a required field'),
+  email: yup
+    .string()
+    .trim()
+    .email('Please provide a valid email')
+    .required('Email is a required field'),
+});
+
+const Form = ({ formClassName }) => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm({ resolver: yupResolver(validationSchema) });
+  const [formState, setFormState] = useState('default');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const onSubmit = async (values) => {
+    setIsLoading(true);
+    try {
+      await fetch('https://formspree.io/f/mdoblnyz', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          ...values,
+        }),
+      });
+      setFormState('success');
+      setTimeout(() => {
+        setFormState('default');
+      }, 3000);
+      setIsLoading(false);
+      reset();
+    } catch (error) {
+      setFormState('error');
+    }
+  };
+
+  return (
+    <section className={formClassName}>
+      <motion.form
+        className="space-y-6"
+        animate={{
+          display: formState === 'success' ? 'none' : 'block',
+          transition: { duration: APPEAR_AND_EXIT_ANIMATION_DURATION },
+        }}
+        method="POST"
+        noValidate
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <div className="relative flex lg:space-x-8 lg:space-y-0 lg:flex-row justify-between flex-col space-y-6">
```

Comment:
```suggestion
        <div className="flex lg:space-x-8 lg:space-y-0 lg:flex-row justify-between flex-col space-y-6">
```

we can get rid of the `relative` class in this case, please see the comment about it below